examples/nuclei_image_segmentation/Nuclei_Image_Segmentation.ipynb

```diff
@@ -0,0 +1,521 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nuclei Image Segmentation Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will implement a UNet to solve Kaggle's [2018 Data Science Bowl Competition](https://www.kaggle.com/c/data-science-bowl-2018). The challenge asks participants to find the location of nuclei from images of cells. In additional to `skorch`, this tutorial uses `matplotlib` for plotting, `torchvision` for image augmentation and Kaggle's offical cli to download data. Please follow [Kaggle's installation and configuration documentation](https://github.com/Kaggle/kaggle-api#installation) to install the kaggle cli. Next, navigate to the [competition page](https://www.kaggle.com/c/data-science-bowl-2018), click on \"Late Submission\" and accept the terms and conditions to get access to the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "from torch.optim import SGD\n",
+    "import torch.nn as nn\n",
+    "from torch.nn.functional import binary_cross_entropy_with_logits\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from skorch.helper import (\n",
```

Comment:
I am not sure if it's good to import unqualified names. This is supposed to be educational notebook, but if everything is imported like this and is used without prefixes, then while reading the notebook it's impossible to tell whether a function is a skorch function or a custom utility. E.g., is `filtered_optimizer()` a skorch function? What about `freeze_layer()`?